inhouse_bot/common_utils/get_server_config.py

```diff
@@ -17,11 +17,8 @@ def get_server_config(server_id: int, session) -> ServerConfig:
     # If server_config doesn't exist for this server_id, we'll create one now and send that back
     server_config = ServerConfig()
     server_config.server_id = server_id
-    server_config.config = {}
-    for key in CONFIG_OPTIONS:
-        server_config.config[key[0]] = False
+    server_config.config = {key: False for (key, value) in CONFIG_OPTIONS}
 
-    session.commit()
```

Comment:
Why the removal of this?

Reply:
Because it could introduce unwanted side effects. Committing halfway during a transaction isn't generally desired unless you want to make sure everything currently in the session can be saved.

The server_config will get committed when the session scope closes